VBZOOMD/VBZOOMD/ZoomRecord.vb

```diff
@@ -182,8 +182,8 @@ Public Class ZoomRecord
         hZOOM_record = 0
         bIgnoreMARCCharacterEncoding = False
         sAssumedMARCCharacterEncoding = ZoomUtil.MARC_8
-        sYAZCharsetFrom = "marc-8"
-        sYAZCharsetTo = "utf-8"
+        sYAZCharsetFrom = ""

```

Comment:
Is empty correct where?

Reply:
yes https://github.com/Notalib/VBZOOMD/blob/fa87cdb148ef515ad906726db4a83949bc75348f/VBZOOMD/VBZOOMD/ZoomRecord.vb#L179